volatility3/framework/plugins/windows/callbacks.py

```diff
@@ -33,27 +35,163 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
             requirements.PluginRequirement(
                 name="ssdt", plugin=ssdt.SSDT, version=(1, 0, 0)
             ),
+            requirements.VersionRequirement(
+                name="poolscanner", component=poolscanner.PoolScanner, version=(1, 0, 0)
```

Comment:
Thanks, I think I forgot to update this when copy-pasting. There should really be three separate requirements - the `handles.Handles` plugin, the `poolscanner.PoolScanner` plugin, and the list of major functions `driverirp.MAJOR_FUNCTIONS`, the last of which I'm not sure how to define a requirement for, since it's not a class in another module, but just a list of strings. I'm going to define it as a `PluginRequirement` for now, but let me know if there's a better way.